test/00.hello_world/test_doc_examples.py

```diff
@@ -11,7 +11,7 @@ def test_vector_add():
     n = 4
 
     # Change this line to ft.optimize(verbose=1) to see the resulting native code
-    @ft.optimize
+    @ft.optimize(verbose=2)
```

Comment:
This test is mean to be exactly the same with the codelet in the document. Please don't change it.